packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -228,30 +249,35 @@ class _GameWidgetState<T extends Game> extends State<GameWidget<T>> {
     // We can use Directionality.maybeOf when that method lands on stable
     final textDir = widget.textDirection ?? TextDirection.ltr;
 
-    return Directionality(
-      textDirection: textDir,
-      child: Container(
-        color: widget.game.backgroundColor(),
-        child: LayoutBuilder(
-          builder: (_, BoxConstraints constraints) {
-            widget.game.onResize(constraints.biggest.toVector2());
-            return FutureBuilder(
-              future: _gameLoaderFutureCache,
-              builder: (_, snapshot) {
-                if (snapshot.hasError) {
-                  if (widget.errorBuilder == null) {
-                    throw snapshot.error!;
-                  } else {
-                    return widget.errorBuilder!(context, snapshot.error!);
+    return Focus(
+      focusNode: widget.focusNode,
+      autofocus: widget.autofocus,
+      onKey: _handleKeyEvent,
+      child: Directionality(
+        textDirection: textDir,
+        child: Container(
+          color: widget.game.backgroundColor(),
+          child: LayoutBuilder(
+            builder: (_, BoxConstraints constraints) {
+              widget.game.onResize(constraints.biggest.toVector2());
+              return FutureBuilder(
+                future: _gameLoaderFutureCache,
+                builder: (_, snapshot) {
+                  if (snapshot.hasError) {
+                    if (widget.errorBuilder == null) {
```

Comment:
nit: save `widget.errorBuilder` to a variable

Reply:
![imagem](https://user-images.githubusercontent.com/6718144/131407038-6154e066-a636-4cbc-b484-94580c8132e2.png)

Reply:
nit: save `widget.errorBuilder` to a variable

Reply:
![imagem](https://user-images.githubusercontent.com/6718144/131501155-c12ad797-5d3b-420b-bea3-25822df245e2.png)

Reply:
![imagem](https://user-images.githubusercontent.com/6718144/131548403-cd4f2dd6-830f-4d9c-a89a-5d4db88a3311.png)